frontend/src/components/CcfriEstimator.vue

```diff
@@ -434,7 +435,7 @@
                 <v-col cols="5" class="estimator-label">
                   <span style="color: #313131"> Select a month (Optional) </span>
                 </v-col>
-                <v-col cols="7" md="1" style="padding-bottom: 0px; padding-top: 16px; padding-left: 0px">
```

Comment:
I think we may have just swapped issues with this fix. I'm noticing the following:
- The slider now shows partial month cards at all resolutions
- Only a single whole month card is visible at Medium
- There's almost too many months available at Small

This screen is pretty complex and finicky so I'm hesitant to change too much. I think we can fix the original issue by just bumping the v-sheet width up by one pixel. See https://github.com/bcgov/EDUC-CCOF/blob/23fc415b5eef4fe6e17e8a4de5f5f955115e0266/frontend/src/components/CcfriEstimator.vue#L457.

Reply:
> The slider now shows partial month cards at all resolutions

I feel like a partial card left or right is not so bad, as it does indicate that there is more to see / scroll to, aside from the "dark arrow" that is.

> Only a single whole month card is visible at Medium

We could tweak the breakpoints to make this more "medium" viewport friendly

> There's almost too many months available at Small

The alternative would be to shrink it down and maybe center align it. It would not match the other input fields in width but it would reduce the amount of noise.

Reply:
Honestly, I just don't feel we should make changes to this established, public-facing tool with going through a proper UI/UX review and that's just not priority right now.

Reply:
Okay, could we backlog this then? I feel like the month selector isn't the only thing that looks a little weird, especially on mobile? I'll work through your suggestions now.

Reply:
@weskubo-cgi reset this branch and merged yours, force pushed. Looks good on local